x-pack/legacy/plugins/apm/server/lib/helpers/es_client.ts

```diff
@@ -147,7 +147,7 @@ export function getESClient(
           `${request.url.pathname} ${JSON.stringify(context.params.query)}`
         );
         console.log(`GET ${nextParams.index}/_search`);
-        console.log(JSON.stringify(nextParams.body, null, 4));
+        console.log(JSON.stringify(nextParams.body, null, 2));
```

Comment:
I changed this to align with Kibana console indentation, which is two spaces. Happy to revert if someone is particularly attached to it of course.

Reply:
👍